server/service/src/requisition/request_requisition/indicator_information.rs

```diff
@@ -46,7 +46,8 @@ pub fn get_indicator_information(
         Some(
             NameFilter::new()
                 .supplying_store_id(EqualFilter::equal_to(store_id))
-                .is_customer(true),
+                .is_customer(true)
+                .is_store(true),
```

Comment:
Indicators only show for customers with stores

Reply:
I think technically this is wrong, but it would be too hard to prove the difference between current mSupply functionality for customer and omSupply